test/bitswap.spec.js

```diff
@@ -5,30 +5,28 @@ const chai = require('chai')
 const dirtyChai = require('dirty-chai')
 const expect = chai.expect
 chai.use(dirtyChai)
-const FactoryClient = require('./ipfs-factory/client')
+
+const DaemonFactory = require('ipfsd-ctl')
+const df = DaemonFactory.create()
 
 describe('.bitswap', function () {
   this.timeout(20 * 1000) // slow CI
-  let ipfs
-  let fc
+  let ipfsd = null
 
   before((done) => {
     this.timeout(20 * 1000) // slow CI
-    fc = new FactoryClient()
-    fc.spawnNode((err, node) => {
+    df.spawn((err, node) => {
       expect(err).to.not.exist()
-      ipfs = node
+      ipfsd = node
```

Comment:
replace this line by: `ipfs = ipfsd.api` and avoid replacing all the other lines.